server/models.py

```diff
@@ -56,7 +56,8 @@ def update_tags(self, category_tags_to_update: Dict) -> None:
         db.session.commit()
 
     def to_json(self) -> Dict:
-        from server.helpers import build_categorised_tags, load_image
+        from server.helpers.tag_helpers import build_categorised_tags
```

Comment:
Get imported every time you call `to_json`
Why not import them once on module load?

Reply:
This was done to resolve a circular import because tag_helpers imports the Tag model (for type hinting...), will spend some time on this to see if it's still the case after the refactor and if I care enough about type hinting to take the relatively big hit here. Also conscious of not spending too much time on this in the interest of moving fast, site isn't super busy so ;)